docs/pt/docs/advanced/custom-response.md

```diff
@@ -0,0 +1,344 @@
+# Resposta Personalizada - HTML, Stream, File e outras
+
+Por padrão, o **FastAPI** irá retornar respostas utilizando `JSONResponse`.
+
+Mas você pode sobrescrever esse comportamento utilizando `Response` diretamente, como visto em [Retornando uma Resposta Diretamente](response-directly.md){.internal-link target=_blank}.
+
+Mas se você retornar uma `Response` diretamente (ou qualquer subclasse, como `JSONResponse`), os dados não serão convertidos automaticamente (memso que você declare um `response_model`), e a documentação não será gerada automaticamente (por exemplo, incluindo o "media type", no cabeçalho HTTP `Content-Type` como parte do esquema OpenAPI gerado).
```

Comment:
Just a typo here:

(memso que você declare um `response_model`)

should be

(*me**sm**o* que você declare um `response_model`)